Dashboard/ViewModels/MainWindowViewModel.cs

```diff
@@ -39,10 +42,15 @@ public partial class MainWindowViewModel : ViewModelBase
         /// </summary>
         public MainWindowViewModel()
         {
+
             _views = new Dictionary<string, (UserControl, ViewModelBase?)>();
             _serviceProvider = DependencyInjection.ConfigureServices();
             Items = new ObservableCollection<ListItemTemplate>(_templates);
             SelectedListItem = Items.First();
+            _footer = new FooterView
```

Comment:
Does the footer change at runtime? Can we not just put the usercontrol directly in the MainWindowView.axaml XML?

Reply:
The Footer could be directly a part of MainWindowView but it would come down to if we want to keep the Navigation system and Footer logic separate for maintaining. I opt'ed to keep them separate just to air on the side of being safe on that regard.

Reply:
I am thinking the view (or control) can remain separate, thats fine. my question was just can we do this statically via the XAML? so no need to dynamically create it in this file

Reply:
I don't think so since the footer needs its own DataContext if we keep them separate from my understanding.

Reply:
Yep, I just tried it myself, I see what you mean. All good :)